Source/DataSources/GeoJsonDataSource.js

```diff
@@ -189,7 +189,7 @@ define([
 
             var description = properties.description;
             if (!defined(description)) {
-                entity.description = new CallbackProperty(createDescriptionCallback(properties, nameProperty), true);
+                entity.description = new CallbackProperty(createDescriptionCallback(describe, properties, nameProperty), true);
```

Comment:
Yes, that's exactly what I had in mind.  Also, due to the automatic wrapping/coercion we do for simple types, they can also return a single string back instead of a `Property` instance and everything will still work.